test/test_ops.py

```diff
@@ -69,6 +70,29 @@
 from torch.utils._pytree import tree_flatten
 from torch.utils._python_dispatch import TorchDispatchMode
 
+
+@enum.unique
+class CtxTestMode(enum.Enum):
```

Comment:
Add a comment describing what this enum is does and how it's intended to be used